src/components/pages/request-form/request-form.component.js

```diff
@@ -0,0 +1,160 @@
+import React from 'react';
+import Paper from '@material-ui/core/Paper';
+import Grid from '@material-ui/core/Grid';
+import { KeyboardDatePicker } from '@material-ui/pickers';
+import FormControl from '@material-ui/core/FormControl';
+import InputLabel from '@material-ui/core/InputLabel';
+import Select from '@material-ui/core/Select';
+import MenuItem from '@material-ui/core/MenuItem';
+import Chip from '@material-ui/core/Chip';
+import Input from '@material-ui/core/Input';
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+
+import useStyles from './request-form.styles';
+
+const RequestForm = () => {
+  const classes = useStyles();
+  const [values, setValues] = React.useState({
+    age: '',
+    name: 'hai',
+  });
+  function handleChangeB(event) {
```

Comment:
You have to use more clear function name and arrow function

Reply:
Fixed.